controller_manager/controller_manager/spawner.py

```diff
@@ -153,10 +158,25 @@ def main(args=None):
     parser.add_argument(
         '--controller-manager-timeout',
         help='Time to wait for the controller manager', required=False, default=10, type=int)
+=======
+        "--controller-manager-timeout",
+        help="Time to wait for the controller manager",
+        required=False,
+        default=10,
+        type=int,
+    )
+    parser.add_argument(
+        "--activate-as-group",
+        help="Activates all the parsed controllers list together instead of one by one."
+        " Useful for activating all chainable controllers altogether",
+        action="store_true",
+        required=False,
+    )
+>>>>>>> 4e78813 (Update spawner to accept controllers list and start them in sequence (#1139))
```

Comment:
```suggestion
    parser.add_argument(
        "--activate-as-group",
        help="Activates all the parsed controllers list together instead of one by one."
        " Useful for activating all chainable controllers altogether",
        action="store_true",
        required=False,
    )
>>>>>>> 4e78813 (Update spawner to accept controllers list and start them in sequence (#1139))
```